tsconfig.json

```diff
@@ -44,7 +44,7 @@
     "declaration": true,
     "removeComments": true,
     "skipLibCheck": true,
-    "noEmitHelpers": true,
+    "noEmitHelpers": false,
```

Comment:
but in case someone does want to use `export { default as xyz }`, we're now covered and the function will be output.